packages/client/src/grpc-retry.ts

```diff
@@ -100,6 +100,7 @@ export function defaultGrpcRetryOptions(options: Partial<BackoffOptions> = {}):
 const retryableCodes = new Set([
   grpc.status.UNKNOWN,
   grpc.status.RESOURCE_EXHAUSTED,
+  grpc.status.INTERNAL,
   grpc.status.UNAVAILABLE,
   grpc.status.ABORTED,
```

Comment:
Wait, if it's supposed to be becoming _retryable_, and this is the list of _retryable_ codes...

Reply:
I can't read. Ignore me.